prep-css-display/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Display Properties</title>
+  <style>
+
+  </style>
+</head>
+
+<body>
+  <div style="border: 4px solid blue;">I am a div and by default I am block, which means I take up all the space I can horizontally.</div>
+  <div style="border: 4px solid blue;">
+    I am a div but I have <span style="border: 4px solid green;" width="100px" height="100px">span elements inside me</span>.
+    Span elements are <span style="border: 4px solid green;" width="100px" height="100px">display inline by default</span>,
+    so they only take up as much as space as they need.
+  </div>
+  <div>
+    <button style="border: 4px solid red;
+height: 40px;
+width: 200px;">Hello!</button>
+    <button style="border: 4px solid red;
+height: 40px;
+width: 200px;">Buttons are</button>
+    <button style="border: 4px solid red;
+height: 40px;
+width: 200px;">Inline Block</button>
```

Comment:
For this exercise we want to put the CSS code in the `<style>` (in the `<head>` element) instead of using a `style` attribute directly on the HTML elements.